test/features/add_user_to_team_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule CascalhoScrumBoard.AddUserToTeamTest do
+  use CascalhoScrumBoard.ModelCase
+
+  alias Ecto.Changeset
+  alias CascalhoScrumBoard.User
+  alias CascalhoScrumBoard.Team
+
+  test "add user to team" do
+    user = %User{email: "taiconery@gmail.com", first_name: "Tacio", last_name: "Nery", password: "1234567890"}
+            |> Repo.insert! |> Repo.preload(:teams)
+
+    team = %Team{name: "Team One", slug: "teamone"} |> Repo.insert! |> Repo.preload(:users)
+
+    changeset = Changeset.change(user) |> Changeset.put_assoc(:teams, [team])
```

Comment:
changeset = Changeset.change(user) |> Changeset.put_assoc(:teams, [team]) |> Repo.update!(changeset)

assert Repo.all(from d in UsersTeams, where: d.user_id == ^user.id AND from t in UsersTeams, where: t. team_id == ^team.id) != []